src/celeritas/global/CoreParams.hh

```diff
@@ -65,11 +65,14 @@ class CoreParams
 
         SPActionRegistry action_reg;
 
+        //! Maximum number of simultaneous threads/tasks per process
+        StreamId::size_type max_streams{1};
+
         //! True if all params are assigned
         explicit operator bool() const
         {
             return geometry && material && geomaterial && particle && cutoff
-                   && physics && rng && init && action_reg;
+                   && physics && rng && init && action_reg && max_streams;
```

Comment:
Since the default assigned value of `max_streams` is `1`, the doc is now slightly off compare to the implementation.  I.e. it sounds like it is now "True if all params are assigned except max_streams, where true means user has either not assigned it or assigned it to a value not zero" :)